### ISSUE TITLE
is it true that temp is saved in R:\ytdltest by default

### ISSUE THREAD

**Issue by candrapersada** [bitingsock/ytdl-preload#5]
and is it possible to use temp=~~/ytdltest
https://mpv.io/manual/master/#paths

**Comment by bitingsock** [comment]
Yes, that is why the description says to change line 21 to what you want. While I could have used LUA's temp file request, in my experience it's implementation is doggy across systems and we aren't using it's actual function anyway. I've not tried relative paths but in theory it should work.

**Comment by jvschiavo** [comment]
> and is it possible to use `temp=~~/ytdltest` or `temp = "~~/ytdltest"` https://mpv.io/manual/master/#paths

I'm using the line `temp=/tmp/ytdl` in the `~/.config/mpv/script-opts/ytdl_preload.conf` file and it works.

I'm using GNU, don't know about Windows.

**Comment by bitingsock** [comment]
that appears to be an issue with your osc script. try without that script enabled

**Comment by candrapersada** [comment]
mpv/scripts/ytdl-preload.lua:35: attempt to index global 'otps' (a nil value)

**Comment by bitingsock** [comment]
oops, typo.
ok try again with the new version or just change that `otps` to `opts`
though that does mean it is not getting a temp directory for you and is using the fallback value anyway

**Comment by bitingsock** [comment]
yes I see. ok now try the new version, tried with the more standard `TMPDIR` for linux

**Comment by candrapersada** [comment]
mpv/scripts/ytdl-preload.lua:23: ')' expected (to close '(' at line 22) near '='

**Comment by bitingsock** [comment]
geez, ok try again

**Comment by candrapersada** [comment]
linux doesn't have `TEMP` or `TMPDIR` in environment variables by default but you can use `/tmp/ytdl`
